0002/README.md

```diff
@@ -0,0 +1,164 @@
+Summarized as follows:
+
+Given a version number MAJOR.MINOR.PATCH, increment the:
+
+* MAJOR version when you make incompatible API changes,
+* MINOR version when you add functionality in a backwards compatible manner, and
+* PATCH version when you make backwards compatible bug fixes.
+
+Additional labels for pre-release and build metadata are available as extensions
+to the MAJOR.MINOR.PATCH format.
+
+### Schedule
+
+Every subproject should be able to follow its own cadence in terms of how often
+or regularly a new release should be cut, as long as it is documented and
+shared with the community.
+
+A new `MINOR` version for Tinkerbell should be released following a proper
+schedule, right now I think every two months sounds reasonable looking at the
+code written and merged in the last couple of weeks. In this way the
+subprojects will have an intrinsic release target if they want to get their new
+code included in the Tinkerbell release.
+
+`PATCH` release for Tinkerbell can be done as required by the subprojects
+bug fixes.
+
+`MAJOR` release right now are not even evaluated because the project is too young.
+
+### GitHub release
+
+GitHub offers the possibility to tag and release code and manifests. Manifests
+can be attached as a release, and they usually contain static binaries or
+resources that can be consumed from a program or from a user without having to
+clone, compile the project itself.
+
+Manifest are crucial, and we should generate and push as manifest everything we
+think can be useful in that sense.
+
+A for examples:
+
+* If a project provides binaries (CLI or daemon) those should be statically
+  compiled and pushed as manifest.
+* We use docker containers and as part of the release process we should write
+  how to pull the correct image from the hub.
+* We have to produce a changelog to provide an easy way for our used to figure
+  out what's new in a release compared with the previous one.
+* If the project produces a  tar.gz or an iso that is too big to get pushed as
+  manifest in GitHub Release we should write in the release the URL of the
+  resource at the right version.
+
+### Architectures and operating systems
+
+We know there are different architectures and Go supports cross compilations.
+When possible we should cross compile binaries at least for:
+
+* Linux
+* macOS
+* Windows
+
+For `arm` and `x86_64`.
+
+### Ownership
+
+The contributors and maintainers for each project have the responsibility for
+the quality of the release itself.
+
+Tinkerbell has a project will probably have its group of interest around release
+management that will take care of it (this is out of scope for this proposal but
+has to be discussed).
+
+### Quality assertion
+
+Every project should be covered by tests: unit tests, functional tests and
+integration as needed.
+
+If all the test suite succeed the project can be released, otherwise tests
+should be fixed first.
+
```

Comment:
Include a review of the documentation (or at least any changes to the documentation) so that it accurately reflects the current features and functionality of the release.

Reply:
Solved